TESTING.md

```diff
@@ -24,15 +24,28 @@ before writing new integration tests, which are slower to run and are not as eas
 Before running tests, make sure your Flutter SDK matches the version that will be used on
 the CI. To update your local flutter version, run:
 
+```shell
+flutter channel main
+flutter upgrade
 ```
-./tool/update_flutter_sdk.sh --local
-```
+
 > Note: Running this command requires that you have followed the [set up instructions](CONTRIBUTING.md#set-up-your-devtools-environment)
 in the DevTools contributing guide regarding cloning the Flutter SDK from GitHub.
```

Comment:
please change this to "...regarding cloning the Flutter SDK from GitHub and adding the `devtools_tool` executable to your PATH."